readme.md

```diff
@@ -168,6 +168,20 @@ Values: `'left'` `'center'` `'right'`
 
 Align the text in the box based on the widest line.
 
+##### title
+
+Type: `string`\
+
+Display a title on the top of the box.
+
```

Comment:
I think it would be useful to show an example of how it will look here. In index.d.ts too.